toolkit/scripts/analysis.mk

```diff
@@ -20,11 +20,15 @@ ifneq ($(build_arch),x86_64)
 # Microsoft repository only exists for x86_64 - skip that .repo file;
 # otherwise package manager will signal an error due to being unable to make contact
 SODIFF_REPO_SOURCES="mariner-official-base.repo"
+SODIFF_REPO_SOURCES_EXTENDED="mariner-official-base.repo mariner-extended.repo"
 else
 SODIFF_REPO_SOURCES="mariner-official-base.repo mariner-microsoft.repo"
+SODIFF_REPO_SOURCES_EXTENDED="mariner-official-base.repo mariner-microsoft.repo mariner-extended.repo"
 endif
 
+
 SODIFF_REPO_FILE=$(SCRIPTS_DIR)/sodiff/sodiff.repo
+SODIFF_REPO_FILE_EXTENDED=$(SCRIPTS_DIR)/sodiff/sodiff-extended.repo
```

Comment:
Could you move these files inside the `$(BUILD_DIR)` directory? That's where we try to keep all files generated during build time, which are meant for internal consumption by the tooling. If it's supposed to be an output for the user, then please use the `$(OUT_DIR)` directory.

Reply:
These were in the scripts_dir from back when this script was made in 1.0. Moved to BUILD_DIR since the placement is pretty arbitrary, and is better there.